integration-cli/docker_api_containers_test.go

```diff
@@ -1134,6 +1134,49 @@ func (s *DockerSuite) TestContainerApiVerifyHeader(c *check.C) {
 	body.Close()
 }
 
+//Issue 14230. daemon should return 500 for invalid port syntax
+func (s *DockerSuite) TestContainerApiInvalidPortSyntax(c *check.C) {
+	config := `{
```

Comment:
Sorry, can we change this to only supply required fields?
This would be `config.Image` and `config.HostCofig.PortBindings` for this test.